spec/controllers/tokens_controller_spec.rb

```diff
@@ -183,8 +191,89 @@
         expect(response.status).to eq 200
       end
 
-      it "revokes the access token" do
-        post :revoke, params: { client_id: client.uid, token: access_token.token }
+      it "does not revoke the access token when token_type_hint == refresh_token" do
```

Comment:
The `token` method was not fully tested so added some specs before I made my changes.